### PR TITLE
Remove dblog and permissions pages since they are not supported at the moment

### PR DESCRIPTION
**Pull request by lauriii**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/199

### CODE REVIEW

**Comment by mattgrill** [comment]
Could we just remove the route overrides from the support module and the route matches on JS side? Seems a bit much to delete everything.

**Comment by lauriii** [comment]
I agree that it is a hard choice to delete these pages from the app now. I think it's better to do delete them completely from the app because we haven't been actively testing those pages and they also don't implement material UI. Maintaining those pages inside the app would add some extra cost with a little to none benefit at this point. Once we are getting more stable, we should just add them back and update to the latest APIs.

**Comment by dawehner** [comment]
Keeping unused code alround is IMHO always worse than removing it. People wonder, try to fix things etc. Getting the page back would be overall less work, I think, when we completely remove it now.

**Comment by dawehner** [comment]
@justafish Do you have an opinion about this?

**Comment by justafish** [comment]
@lauriii the issue you linked is just for removing the dblog page, which I think is fine. Permissions works and is definitely a UX improvement even if it doesn't use Material UI yet, so let's leave that in.

**Comment by mattgrill** [comment]
@justafish - I think we did just fix the dblog page.

> Keeping unused code alround is IMHO always worse than removing it.

I agree. I guess I don't have a real problem with removing some stuff for the sake of the demo.

File: admin_ui_support/admin_ui_support.routing.yml [hunk 1]
```diff
@@ -1,48 +1,4 @@
 # Routes controlled by the React App.
-admin_ui_support.user.permissions:
```
Comment (dawehner) [inline admin_ui_support/admin_ui_support.routing.yml]: This certainly removes the permissions page as well

**Comment by dawehner** [comment]
> I agree. I guess I don't have a real problem with removing some stuff for the sake of the demo.

How about writing a test to ensure it always works?

File: admin_ui_support/admin_ui_support.routing.yml (hunk 1 above)
Comment (mattgrill) [reply admin_ui_support/admin_ui_support.routing.yml]: Maybe we can just remove the dblog component here and have an issue to update the permissions/roles pages to the material ui design?

Comment (dawehner) [reply admin_ui_support/admin_ui_support.routing.yml]: Sounds sensible for me!

**Comment by dawehner** [comment]
@lauriii Are you planning to come back to that or should we tag this issue as "good first issue"?

**Comment by justafish** [comment]
Looks like no one's working on this, so let's close for now and address when we do our clean up/housekeeping during around Drupal Europe

**Pull request closed without merging**